build.gradle

```diff
@@ -542,7 +542,7 @@ project(':iceberg-mr') {
 
   test {
     // testJoinTables / testScanTable
-    maxHeapSize '1500m'
+    maxHeapSize '2500m'
```

Comment:
Why was this needed? Additional tasks because of partitioning?

Reply:
Yeah. The extra tasks eat more memory :(